### DISCUSSION TITLE
Want to get old Allied Leisure HoeDown machine working using MPF/COBRAPin looking for advice

### DISCUSSION THREAD

**Discussion by DrFistington** [missionpinball/missionpinball#158] [Q&A]
I have an old Allied Leisure Hoe Down machine that's in physically great shape, but the power supply started to have issues years ago which led to the MB being completely fried.  Finding a replacement Thunderbolt Rev3 board for the machine is pretty much impossible.  I could MAYBE get another broken board for $400, but at this point I would rather just get it working by replacing the board with a small linux box running MPF and the Cobra Pin controllers.

I guess my main questions are; does this sound like it will be feasible to do with the Cobra Pin boards?  Also, does anyone know of any similar restoration projects that have the machine code setup on github that I could study?

**Comment by avanwinkle** [comment]
Sounds like a great project! However note that Cobra Pin (or any homebrew platform) is just an interface for the machine and MPF is just a platform for running game rules, so you'd be responsible for creating all of the rules for the game from scratch. If there are any sounds triggered mechanically you'll have the ability to trigger those, but sounds and lights and everything else will be up to you to write by hand.

A lot of Cobra folks are active on the MPF google group, so any questions you have about the boards would be best asked there. Good luck!

https://groups.google.com/g/mpf-users

**Comment by DrFistington** [comment]
Good to hear, that's exactly what I was looking for. This machine is old
enough that it only has sounds from the 3 chimes and the knocker, and the
rules aren't too advanced, but it will be fun trying to rewrite them. Is
the Cobra pin able to interface with LED scoreboards as well?
